src/families/filecoin/test-dataset.ts

```diff
@@ -96,26 +104,6 @@ const filecoin: CurrenciesData<Transaction> = {
             warnings: {},
           },
         },
-        {
-          name: "Amount Required",
-          transaction: fromTransactionRaw({
-            family: "filecoin",
-            method: 1,
-            version: 1,
-            nonce: 100,
-            gasFeeCap: "1000",
-            gasLimit: 10,
-            gasPremium: "10000",
-            recipient: ACCOUNT_1,
-            amount: "0",
-          }),
-          expectedStatus: {
-            errors: {
-              amount: new AmountRequired(),
-            },
-            warnings: {},
-          },
-        },
```

Comment:
Any reason for removing that one? Testing `AmountRequired` seems to make sense.

Reply:
I put the test back where it was!